tiledb/sm/serialization/query.cc

```diff
@@ -1696,9 +1719,7 @@ Status query_from_capnp(
             // eventually update the query).
             attr_copy_state->offset_size += fixedlen_size_to_copy;
             attr_copy_state->data_size += varlen_size;
-            if (nullable)
-              attr_copy_state->validity_size += validitylen_size;
-
+            attr_copy_state->validity_size += validitylen_size;
```

Comment:
why?

Reply:
Initially I was just removing the one line if statement, but I realized that validitylen_size will be zero if it's not nullable, so I left the if statement out entirely. I can add this back though if it's a concern. :+1: